cypress/e2e/main-inputs-interaction/city-input-interaction.cy.js

```diff
@@ -0,0 +1,29 @@
+describe('City input interaction', () => {
+  it('City input must have typed value', () => {
+    cy.visit('/');
+    cy.get('[data-cy=main-input]:nth-child(1) input').as('city-input');
+    cy.get('@city-input').type('Paris');
+    cy.get('@city-input').should('have.value', 'Paris');
+  });
+
+  it('Url must have search param city with typed value', () => {
+    cy.visit('/');
+    cy.get('[data-cy=main-input]:nth-child(1) input').as('city-input');
+    cy.get('@city-input').type('Paris');
+    cy.url().should('include', 'city=Paris');
+  });
+
+  it('Input must have typed value after page reload', () => {
+    cy.visit('/');
+    cy.get('[data-cy=main-input]:nth-child(1) input').as('city-input');
+    cy.get('@city-input').type('Paris');
+    cy.reload();
+    cy.get('@city-input').should('have.value', 'Paris');
```

Comment:
логические отступы и в тестах полезны
```suggestion
    cy.visit('/');
    
    cy.get('[data-cy=main-input]:nth-child(1) input').as('city-input');
    cy.get('@city-input').type('Paris');
    
    cy.reload();
    
    cy.get('@city-input').should('have.value', 'Paris');
```

Reply:
и тест выглядит крайне слабеньким. Мы даже не проверили, что там изначально в инпуте